### PR TITLE
move rhel repos to downstream.yaml

### PR DESCRIPTION
**Pull request by rakeshgm**
remove unsupported versions from this branch and added rhel_repos to be enabled for luminous. 

Signed-off-by: rakeshgm <rakeshgm@redhat.com>

### CODE REVIEW

**Review by sunilkumarn417 — APPROVED** [review]
LGTM

**Comment by rakeshgm** [comment]
this is based on commits, https://github.com/ceph/teuthology/commit/c32831cde8222926ea77341225c7a405d6c90ed6. which will be sent to upstream

File: qa/rh/downstream.yaml [hunk 1]
```diff
@@ -75,3 +45,10 @@ pkgs:
     - python-rbd
     - python-cephfs
     - rbd-mirror
+
+rhel_repos:
+  '7':
+    - rhel-7-server-rpms
+    - rhel-7-server-optional-rpms
```
Comment (v26i) [inline qa/rh/downstream.yaml:52]: same https://github.com/red-hat-storage/ceph/pull/116#discussion_r461474745 , may be earlier versions ( current unsupported versions like ceph 1.x,2.x might have used optional-rpms but definitely rh-luminous dont need it as per doc)

Comment (rakeshgm) [reply qa/rh/downstream.yaml:52]: correct, but we install many packages based on the tests and some might be from optional-rpms. these packages are from install task.

**Pull request merged**